README.md

```diff
@@ -98,6 +98,13 @@ are registered with the container.
 
 ## Configuration
 
+If you use the [zend-component-installer](https://github.com/zendframework/zend-component-installer) 
+the factories are configured automatically for you when requiring this package
+wit composer. Without the component installer, you need to 
+include the `[ConfigProvider](src/ConfigProvider.php)` in your 
+`[config/config.php](https://github.com/zendframework/zend-expressive-skeleton/blob/master/config/config.php)`. 
```

Comment:
Fix position of "`" with above links, it should be inside the square brackets, have a look how it looks here:
https://github.com/xtreamwayz/zend-expressive-twigrenderer/tree/feature/optional-twig-extension#configuration